### PR TITLE
PERF: I was looking into this file and a saw some duplicate strings so i cr…

### PR DESCRIPTION
**Pull request by JoaoLemos00**
…eated a constant for those strings"

- [ ] closes #xxxx (Replace xxxx with the GitHub issue number)
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

File: .scannerwork/report-task.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+projectKey=frame
```
Comment (jbrockmendel) [inline .scannerwork/report-task.txt:1]: are these files needed?

File: pandas/core/frame.py [hunk 2]
```diff
@@ -466,7 +466,14 @@
...
 """
-
+#Define Constants
+#-----------------------------------------------------------------------
+DISPLAY_MAX_ROWS = "display.max_rows"
+DISPLAY_MIN_ROWS = "display.min_rows"
+DISPLAY_MAX_COLUMNS = "display.max_columns"
+DISPLAY_MAX_COLDWIDTH = "display.max_colwidth"
+DISPLAY_SHOW_DIMENSIONS = "display.show_dimensions"
+MODE_DATA_MANAGER = "mode.data_manager"
```
Comment (jbrockmendel) [inline pandas/core/frame.py:476]: whats the upside here?  these aren't even fewer characters

**Review by mroeschke — CHANGES_REQUESTED** [review]
As is I am -1 on these changes except if there's a noticeable performance difference
